generator/test_case_generator.go

```diff
@@ -105,7 +105,6 @@ var testTypeToTestConfig = map[string][]testConfig{
 	*/
 	"ec2_mac": {
 		{testDir: "../../../test/feature/mac"},
-		{testDir: "../../../test/run_as_user"},
```

Comment:
We don't need this. macOS has to use validator so we can avoid starting up macOS instances that are more than needed due to limit macOS quota. Instead we manually run the run_as_user go test on one host,.